packages/cli/src/utils/string.js

```diff
@@ -0,0 +1,13 @@
+function splitFileFromPath(filePath, extension = 'js') {
+  const destParts = filePath.split('/');
+  const filename = destParts
+    .splice(-1, 1)
+    .concat(`.${extension}`)
+    .join('');
+  const dirPath = destParts.join('/');
+  return [filename, dirPath];
```

Comment:
This feels a little backwards to me, I think i'd expect `[dirPath, filename]` since that's the way it would be written out: `src/triggers/foo.js`. nbd either way though.

Reply:
That's a fair point. For some reason I was thinking we were more likely to want the file name vs the dir.

Reply:
I think that's a fair assumption. honestly, could `return { filename, dirPath }` and then we can always grab exactly what we want.

Reply:
I thought about returning an object, but I also like the idea of returning a tuple to make naming the values whatever you want easier—in the case where you call this function twice in the same scope. 

```js
const { dirPath: dirA, filename: fileA,  } = split(pathA)
const { dirPath: dirB, filename: fileB,  } = split(pathB)

// vs

const [dirA, fileA] = split(pathA)
const [dirB, fileB] = split(pathB)
```

And if we wanted just filename
```js
const [,fileName] = split(path)
```

In the end, it's a minor tradeoff either way—not sure if this function will even get used outside of `scaffold`.